dbt_semantic_interfaces/implementations/metric.py

```diff
@@ -14,6 +16,8 @@
     PydanticWhereFilterIntersection,
 )
 from dbt_semantic_interfaces.implementations.metadata import PydanticMetadata
+from dbt_semantic_interfaces.protocols import ProtocolHint
+from dbt_semantic_interfaces.protocols.metric import MetricConfig
```

Comment:
Nit: We should add `MetricConfig` to the protocols `__init__.py` so that we can import it from the `protocols` path.